blt-project/blt/sites/example.yml

```diff
@@ -0,0 +1,21 @@
+# @file
+# example.yml
+# Site definition file for remote and local environment information.
+#
+# The filename of this file should correspond to docroot/sites/[site-name]
+# (e.g. default.yml for docroot/sites/default). You should create one file in
+# this directory for each site running against your codebase.
+aliases:
```

Comment:
I've made this change and moved the configuration this file supplies to `phing/example.multisite.yml` so it doesn't have to be explicitly ignored during initialization.